### ISSUE TITLE
server restart spam

### ISSUE THREAD

**Issue by walker-dev** [Orange1861/Improved-Villagers#13]
Every server restart I get like 1300 lines in 1 sec telling me [22:03:50] [Server thread/WARN]: Skipping Piece with id Village, see log: [latest.log](https://gist.github.com/boosterdev/29cc00981d324ddf8cb15cc26c882ba9)
other then that no problems occur

**Comment by Orange1861** [comment]
Yes, this is an known bug that means that MC is skipping bad village gen from this mod. It is safe to ignore it and it may be be fixed by disabling cities (not villages, but cities) in the config file. Villages are next in my agenda to fix after completion of the Gui changes.

**Comment by walker-dev** [comment]
Great looking forward :+1:
